package.json

```diff
@@ -37,9 +37,10 @@
   "homepage": "https://github.com/azu/nlp-pattern-match",
   "devDependencies": {
     "husky": "^0.14.3",
-    "lerna": "^2.11.0",
+    "lerna": "^3.22.1",
     "lint-staged": "^7.2.0",
-    "prettier": "^1.13.5"
+    "prettier": "^1.13.5",
+    "typescript": "4.1.3"
```

Comment:
Does this install multiple typescript versions into the project?
It seems that yarn.lock has typescript 4.1.3 and typescript@^2.5.3.(These are not deduped)
Probably, you need to update each `typescript`s via `yarn upgrade-interactive --latest`.

Reply:
Yes, I updated it.